### PR TITLE
Command line option for the step size

### PR DESCRIPTION
**Pull request by lochel**

### CODE REVIEW

File: src/OMSimulatorLib/Flags.h [hunk 1]
```diff
@@ -158,6 +165,7 @@ namespace oms
       {"--solver", "", "Specifies the integration method (euler, [cvode])", re_default, Flags::Solver, false},
       {"--solverStats", "", "Adds solver stats to the result file, e.g. step size; not supported for all solvers (true, [false])", re_bool, Flags::SolverStats, false},
       {"--startTime", "-s", "Specifies the start time", re_double, Flags::StartTime, false},
+      {"--stepSize", "", "Set step size", re_stepSize, Flags::StepSize, false},
```
Comment (AnHeuermann) [inline src/OMSimulatorLib/Flags.h:168]: Add a hint on how to use `--stepSize` with minimum and maximum step size.

File: src/OMSimulatorLib/Flags.cpp [hunk 2]
```diff
@@ -391,6 +396,32 @@ oms_status_enu_t oms::Flags::StartTime(const std::string& value)
...
+
+  return result;
+}
+
+oms_status_enu_t oms::Flags::StepSize(const std::string& value)
+{
+  std::vector<std::string> stepSize = split(value, ',');
+  if (stepSize.size() > 1)
+  {
+    GetInstance().initialStepSize = atof(stepSize[0].c_str());
```
Comment (AnHeuermann) [inline src/OMSimulatorLib/Flags.cpp:416]: We could add some basic error checks, e.g. no negative numbers for steps sizes.
But that was the only way I could find to break this flag.

File: src/OMSimulatorLib/Flags.h (hunk 1 above)
Comment (lochel) [reply src/OMSimulatorLib/Flags.h]: done.

File: src/OMSimulatorLib/Flags.cpp (hunk 2 above)
Comment (lochel) [reply src/OMSimulatorLib/Flags.cpp]: done

**Pull request merged**